### PR TITLE
lab-06-chris

### PR DESCRIPTION
**Pull request by ChristopherSClosser**

### CODE REVIEW

File: lab-06-chris/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+paragraph about me:I am 36, Married for 17 years. I have two teenage boys 13 and 16. I joined the Air Force straight out of high school, spent 6 years as a fuels journeyman.I enjoy being outdoors, I love knowing the inner workings of everything, finish carpentry, and basically making stuff with my hands.
```
Comment (devonhackley) [inline lab-06-chris/README.md:1]: Nice job with the read me. Generally the read me is more about the application and less about the dev. You'll want to discuss what you application is.

File: lab-06-chris/server.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  let userEntry = string.split(' ').shift().trim();
+
+  client.nickName = userEntry;
+  pool.forEach(c => c.socket.write(`User: ${oldName}\nChanged name to: ${client.nickName}\n`));
+});
+
+ee.on('@DM', (client, string) => {
+  let fromUser = client.nickName;
+  let msg = string.split(' ').slice(1).join(' ').trim();
+  let holdString = string.split(' ');
```
Comment (devonhackley) [inline lab-06-chris/server.js]: So whats the use of holdString here?

File: lab-06-chris/server.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  let userEntry = holdString.shift().trim();
+
+  function findClient(client) {
+    return client.nickName === userEntry;
+  }
+
+  let clientObj = pool.find(findClient);
```
Comment (jurrish) [inline lab-06-chris/server.js:37]: This was a really cool use of .find.  When you guys find quirks and doc methods and implement them = win.

File: lab-06-chris/test/client-test.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+  describe('Someone has joined', () => {
+    it('Should notify all users that Someone new has connected', done => {
+      let user = net.connect({port: 3000}, () => {
+        user.once('data', (data) => {
+          expect(data.toString()).to.include('has connected');
+        });
+        user.end();
+        done();
+      });
```
Comment (jurrish) [inline lab-06-chris/test/client-test.js:26]: test the rest of your commands for full test credit!

**Pull request merged**